drizzlepac/hlautils/se_source_generation.py

```diff
@@ -178,7 +183,7 @@ def create_sextractor_like_sourcelists(source_filename, catalog_filename, param_
     # Regenerate the source catalog with presumably now only good sources
     seg_cat = source_properties(imgarr_bkgsub, segm, filter_kernel=kernel, wcs=imgwcs)
```

Comment:
From the photutils docs:  If you input the background image that was subtracted from the data into the background keyword of source_properties(), the background properties for each source will also be calculated.  This would be done by adding 'background=bkg.background, if I interpret everything correctly.

Reply:
OK